pkg/handlers/systems.go

```diff
@@ -11,12 +12,10 @@ import (
 )
 
 type fetchSystems func() (models.SystemShorts, error)
-type marshal func(interface{}) ([]byte, error)
```

Comment:
This was nice in theory, but doesn't provide the value of its abstraction. Especially, with using decode for incoming data. Happy to chat more about why.